scripts/mbedtls_dev/asymmetric_key_data.py

```diff
@@ -49,14 +49,80 @@ def construct_asymmetric_key_data(src) -> Dict[str, Dict[int, bytes]]:
 ## pairs consist of a private key and its public key.
 #pylint: disable=line-too-long
 ASYMMETRIC_KEY_DATA = construct_asymmetric_key_data({
+    'ECC(PSA_ECC_FAMILY_SECP_K1)': {
+        192: ("297ac1722ccac7589ecb240dc719842538ca974beb79f228",
+              "0426b7bb38da649ac2138fc050c6548b32553dab68afebc36105d325b75538c12323cb0764789ecb992671beb2b6bef2f5"),
+        224: ("0024122bf020fa113f6c0ac978dfbd41f749257a9468febdbe0dc9f7e8",
```

Comment:
@gilles-peskine-arm  I have trouble reading the code use to generate the data, but the `00` at the beginning seems spurious. This results in a key that looks 232-bit long based on the number of bytes in the string, now causing issues in some update tests. Can you confirm that was not intentional and we can just remove the leading `00`?

Reply:
secp224k1 has 224-bit coordinates but 225-bit private keys. The nominal key size in PSA is the private key size, hence 225. So the data looks correct to me. The `00` at the beginning would be `01` for a small fraction of private keys.

[Our PSA code doesn't handle sec224k1](https://github.com/Mbed-TLS/mbedtls/issues/3541) so we don't actually need this data for positive PSA tests. We do need not-supported test cases though (we want a test failure if we accidentally allow the creation of a secp224k1 key).

Reply:
Oh right, well I'm glad I asked then :)

We noticed this because some recent changes in https://github.com/Mbed-TLS/mbedtls/pull/8664 (not merged yet, fortunately) caused failures with the not-supported test cases. Removing the leading `00` fixed the failures but was probably the wrong way.

I see you just committed to reviewing 8664, so that's perfect: you'll be able to comment on that change there then.